source/overview/security.rst

```diff
@@ -36,17 +36,17 @@ Transmission Security
 
    - Mattermost supports TLS encryption using AES-256 with 2048-bit RSA on all data transmissions across both LAN and internet.
    - Connections to Active Directory/LDAP can be optionally secured with TLS or stunnel (E10).
-   - Encryption-at-rest is available through hardware and software disk encryption solutions applied to the Mattermost database, which resides on its own server within your infrastructure. To support end user search and compliance reporting of message histories, Mattermost does not offer encryption within the database. 
+   - Encryption-at-rest is available through hardware and software disk encryption solutions applied to the Mattermost database, which resides on its own server within your infrastructure. To support end user search and compliance reporting of message histories, Mattermost does not offer encryption within the database.
    - Option to `exclude message contents from push notifications <https://docs.mattermost.com/administration/config-settings.html#push-notification-contents>`_ to comply with strict compliance policies, such as US HIPAA standards.
-   - Ability to exclude or include the `contents of messages in push notifications <https://docs.mattermost.com/administration/config-settings.html#push-notification-contents>`_ to avoid disclosure on locked mobile screens, and via relay servers from Apple and Google when sending notifications to iOS or Android mobile apps (relevant to compliance standards such as HIPAA) 
+   - Ability to exclude or include the `contents of messages in push notifications <https://docs.mattermost.com/administration/config-settings.html#push-notification-contents>`_ to avoid disclosure on locked mobile screens, and via relay servers from Apple and Google when sending notifications to iOS or Android mobile apps (relevant to compliance standards such as HIPAA)
 
 Integrity & Audit Controls
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-   - By default, Mattermost stores a complete history of messages, including edits and deletes, along with all files uploaded. User interface actions for "deleting" messages and channels remove the data only from the user interface; the data is retained within your database.
+   - By default, Mattermost stores a complete history of messages, including edits and deletes, along with all files uploaded. User interface actions for "deleting" messages and channels remove the data only from the user interface; the data is retained within your database. Furthermore, you can turn off the ability for users to edit their messages after they're posted.
```

Comment:
Maybe we can go with something like: 

"If your compliance guidelines require it, you can optionally turn off the ability to edit and delete messages after they are posted." 

0/5 - I think the idea is to explain why people would want to turn off edit/delete, and we should probably include the delete in there as well as edit.